File: iniciante/1002/1002.c
// 1002 - Area do Circulo - beecrowd

#include <stdio.h>
#define pi 3.14159

void calc(double r)
{
    double area;

    area = pi * (r * r);
    printf("A=%.4lf\n", area);
}

int main(void)
{
    double R; scanf("%lf", &R);

    calc(R);

    return 0;
}

File: iniciante/1002/1002.js
// 1002 - Area do Circulo - beecrowd

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const n = 3.14159;
const raio = parseFloat(lines[0]);

const area = n * Math.pow(raio, 2);

console.log(`A=${area.toFixed(4)}`);
